src/Loggers/ConsoleLogger.php

```diff
@@ -57,6 +71,10 @@ public function log($level, $message, array $context = []): void
         }
         $stop = !empty($color) ? static::COLOR_STOP : '';
 
+        if (!in_array($level, self::UNCRITICAL_LOG_LEVELS, true)) {
```

Comment:
I don't like the current naming of `UNCRITICAL_LOG_LEVELS` and `loggedError`.

Will rename to `containsMessageAboveNotive`.

Reply:
done